mParticle-Kochava/MPKitKochava.m

```diff
@@ -2,7 +2,11 @@
 #import "MPKochavaSpatialCoordinate.h"
 #import "mParticle.h"
 #import "MPKitRegister.h"
+#if defined(__has_include) && __has_include(<KochavaTracker/KochavaTracker.h>)
+#import <KochavaTracker/KochavaTracker.h>
+#else
 #import "KochavaTracker.h"
+#endif
```

Comment:
Can you also update the lines below where ternary syntax is used? Kochava fields are BOOLs so we should be setting as YES/NO not @YES/@NO